### ISSUE TITLE
Sim not recognized in slot 1 with sd card inserted.

### ISSUE THREAD

**Issue by lean6sigma** [HelloVolla/ubuntu-touch-beta-tests#130]
### Is this a new report?

Yes

### Device model

Volla Phone 22 (2022)

### OS version

Ubuntu 16.04.7 LTS

### Describe the bug

Sim in slot 1 only is recognized if there are no sd card in.
Sim in slot 2 is recognized with or without a sd card.
I only tested with one sim card only, I don't have a second diferent one to test with both slots at the same time.

### Steps to reproduce the bug

1: Insert sd card.
    Insert sim in slot 1
    Boot

2: Remove sd card
    Mantain sim in slot 1
    Boot

3: Insert sd card.
    Insert sim in slot 2
    Boot

4:Remove sd card
    Mantain sim in slot 2
    Boot

### Expected behaviour

1: Sim not recognized
2: Sim recognized
3: Sim recognized
4: Sim recognized

### Additional context

![photo_2022-08-12_06-27-02](https://user-images.githubusercontent.com/28131381/184467166-3eec1b88-6566-4a2d-af99-bb745bd31648.jpg)

**Comment by JamiKettunen** [comment]
@lean6sigma What you've filled in as the `OS version` here is your `Serial` from the About screen. Since you didn't seen to find it I assume this is Ubuntu Touch as a dual-boot secondary system installed to an SD card? If that's the case there unfortunately isn't yet a way to find the image version for that which was installed so this issue template actually needs to be updated.

**Comment by JamiKettunen** [comment]
You mention **removing SD card** as well so it can't be dual-boot, please take another look in the System Settings `About` screen to find the OS version (it should be `16.04` something) and if you can't find it let me know as well.

There is a problem with the modem daemon `mtkfusionrild` crashing on boot sometimes for which the cause is not entirely understood yet, but a workaround to restart it in case it does has been added. It's currently unclear when this patch will be available for `Stable` channel users however, `Developer` update channel has this change already for now.

You could also try rebooting a few times with SD card inserted and SIM in slot 1.

**Comment by lean6sigma** [comment]
It only says Volla Phone 22 below UBUNTU at top of the screen. Is the OS version that came with the new phone last week from Volla.

**Comment by JamiKettunen** [comment]
@lean6sigma Please don't post your IMEI numbers, I understand the OS version isn't shown in System Settings > About screen.

**Comment by lean6sigma** [comment]
In the Ubuntu Touch Tweak Tool it says that is a Ubuntu 16.04.7 LTS. I already corrected in the inicial form.
BTW: What you refer as hapning randomly "There is a problem with the modem daemon mtkfusionrild crashing on boot sometimes for which the cause is not entirely understood yet..." it also hapned with me. Sometime I need too repeat the reboot for the sim appears functioning.

**Comment by JamiKettunen** [comment]
@lean6sigma Does rebooting similarly also fix the SIM in slot 1 not recognized with SD card inserted? If yes I can update the issue title to something more appropriate.

**Comment by lean6sigma** [comment]
No. For the case I report is like I report, happens all the times.

What in 2 or 3 times hapned was that for some reason after I needed to reboot the sim didn't appear functuning. But after rebooting again it works.

**Comment by lean6sigma** [comment]
In Ubports forum someone said that some card brands were problematic. So I tried with onother card and the slot 1 got recognized. After that I tried with the sd card that originaly didn't let slot 1 to be recognized and surprise the slot1 became recognizes as well.
I repeated the procedure and everything got working normally. 
The sd that originally was problematic was Sandisk Extreme PRO 64GB.
The sd that I used to try a diferent brand was Integral 32 GB.
Now the sim in slot 1 is recognizes with both cards.
